src/lib.rs

```diff
@@ -807,6 +808,30 @@ pub unsafe trait KnownLayout {
     }
 }
 
+/// Efficiently produces the [`TrailingSliceLayout`] of `T`.
+#[must_use = "has no side effects"]
```

Comment:
No longer necessary now that this isn't a method on `KnownLayout`, and thus isn't public.

Reply:
https://github.com/google/zerocopy/compare/101bbb6a420aac10d8bf4312305e10f919a3a294..ddb594e5907f90885faebb3e27c6eeb2a1797a0a#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759L812